frontend/src/metabase/core/components/TabButton/TabButton.tsx

```diff
@@ -46,22 +48,24 @@ export interface TabButtonProps<T> extends HTMLAttributes<HTMLDivElement> {
   value: T;
   showMenu?: boolean;
   menuItems?: TabButtonMenuItem<T>[];
-  onEdit?: ChangeEventHandler<HTMLInputElement>;
-  onFinishEditing?: () => void;
-  isEditing?: boolean;
+  onRename?: ChangeEventHandler<HTMLInputElement>;
```

Comment:
Refactored to use the word `rename` instead of `edit` to avoid confusion with the `isEditing` prop inside the dashboard, plus it's more descriptive.